packages/svelte-query/src/useQueryClient.ts

```diff
@@ -0,0 +1,14 @@
+import type { QueryClient } from '@tanstack/query-core'
+import { client } from './store'
+
+export function useQueryClient(): QueryClient {
+  let queryClient!: QueryClient
+  client.subscribe((val) => {
+    queryClient = val
+    queryClient.mount()
+    return () => {
+      queryClient.mount()
```

Comment:
```suggestion
      queryClient.unmount()
```

Reply:
In hindsight, this whole function could be replaced with `return get(client)`. (I'm still trying to understand why this needs to be a store?)

Reply:
@ivanvanderbyl, you're right, this does not have to be a store.
However, it has to be mounted/unmounted.

I think the best solution is to use `getContext`/`setContext` svelte API.
What do you think?